src/functions.js

```diff
@@ -0,0 +1,95 @@
+import UpdateStorage from './Modules/updateStorage.js';
+import { addTask } from './Modules/addRemove.js';
+
+const displayTasks = (taskInject) => {
+  const tasks = JSON.parse(localStorage.getItem('storedStTask')) || [];
+;
+  taskInject.innerHTML = '';
+  tasks.forEach((task) => {
+    const li = document.createElement('li');
+    li.classList.add('li');
+    li.setAttribute('id', task.index);
+    taskInject.appendChild(li);
+    const span = document.createElement('span');
+    span.classList.add('task-wrapper');
+    li.appendChild(span);
+    const check = document.createElement('input');
+    check.classList.add('task-check');
+    check.type = 'checkbox';
+    check.checked = task.completed;
+    span.appendChild(check);
+    const label = document.createElement('label');
+    label.classList.add('dat-task');
+    label.innerText = task.description;
+    span.appendChild(label);
+    const div = document.createElement('div');
+    li.appendChild(div);
+    const edtBtn = document.createElement('button');
+    div.appendChild(edtBtn);
+    const dltBtn = document.createElement('button');
```

Comment:
I would recommend refactoring the creation of buttons (edtBtn and dltBtn) by extracting it into a reusable function like createButton. This will help eliminate repetition and enhance code maintainability. Keep up the good work!

Reply:
I've extracted the creation of buttons into a reusable function createButton